src/vl-wizard.js

```diff
@@ -0,0 +1,120 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+import '/node_modules/vl-ui-progress-bar/dist/vl-progress-bar-all.js';
+import '/lib/wizard.js';
+import './vl-wizard-pane.js';
+
+/**
+ * VlWizard
+ * @class
+ * @classdesc Gebruik de wizard om de gebruiker door een proces met meerdere stappen te begeleiden. Een wizard laat je toe om een complex proces op te splitsen in hapklare acties. Een wizard maakt het ook mogelijk om opties afhankelijk te maken van de keuzes in een voorgaande stap.
+ * 
+ * @extends VlElement
+ * 
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-wizard/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-wizard/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-wizard.html|Demo}
+ * 
+ */
+export class VlWizard extends VlElement(HTMLElement) {
+    constructor() {
+        super(`
+            <style>
+                @import '/src/style.css';
+
+                .vl-wizard__panes {
+                    overflow: hidden;
+                }
+            </style>
+            <section class="vl-wizard" data-vl-wizard>
+                <header class="vl-wizard__heading" role="none">
+                    <slot name="title"></slot>
+                    <slot name="header"></slot>
+                </header>
+                <vl-progress-bar></vl-progress-bar>
+                <div class="vl-wizard__panes">
+                    <slot name="panes"></slot>
+                </div>
+            </section>
+        `);
+
+        this.__callback = {
+            callbackFn: Promise.resolve(true)
+        };
+    }
+
+    async connectedCallback() {
+        await this._processPanes();
+        this._observeProgressBarClick();
+        this._dress();
+    }
+
+    /**
+     * Geeft de callback functie die bepaalt of er naar de volgende/vorige pagina mag genavigeerd worden.
+     * 
+     * @return {Function}
+     */
+    get callback() {
```

Comment:
Ik vind de naamgeving nog wat te generiek. Uit de naam blijkt nu niet echt waarvoor dit dient.

Reply:
Ik sta open voor alternatieven, maar nu heb ik de naamgeving van Webuniversum aangehouden.

Reply:
`readyToStep`, bijvoorbeeld. 
En als je promise dan een object zou teruggeven, kan je dat vertalen naar de promise die webuniversum verwacht: 

```
readyToStep.then(status => {
  if(status.next && status.previous) 
    return true;
  else if(status.next)
    return undefined;
});
```

Of iets in die aard.

Reply:
Ik heb de naamgeving van de param aangepast, misschien verzacht dat de pijn?